doc/getstarted/basic_usage/index_en.rst

```diff
@@ -70,20 +68,19 @@ Now that everything is ready, you can train the network with a simple command li
         paddle train --config=trainer_config.py --save_dir=./output --num_passes=30
  
 
-This means that PaddlePaddle will train this network on the synthectic dataset for 30 passes, and save all the models under path ``./output``. You will see from the messages printed out during training phase that the model cost is decreasing as time goes by, which indicates we are getting a closer guess.
+This means that PaddlePaddle will train this network on the synthetic dataset for 30 passes, and save all the models under the path :code:`./output`. You will see from the messages printed out during training phase that the model cost is decreasing as time goes by, which indicates we are getting a closer guess.
```

Comment:
Here PaddlePaddle will train this network on the synthetic dataset for 30 passes, save all the models under the path :code:`./output`. During training, from the log messages, the model cost is decreasing along with the increasing number of passes, which indicates we are getting the closer "guess" of the learning parameters.